IntegrationTests/DualFPGA/firmware/cfg/vsim_f2.dep

```diff
@@ -0,0 +1,17 @@
+#don't include cm_v2_p1_2 since without decoder, it will have unresolved tcds2 dependencies
+@fpga_position = "2"
+@fpga_speed_grade = "-2-e"
+
+include payload_f2.dep
+setup ../ucf/vsim_f2.tcl
+
+include -c emp-fwk:boards/testbench top.dep
+
+src tb_decl_f2.vhd
+src emp_project_decl_f2.vhd 
+src -c emp-fwk:components/links/slink emp_slink_types.vhd
+
+src -u sim ../scripts/pre_msim.tcl
+
+include -c emp-fwk:boards/apollo/cm_v2/vu13p device.dep packages_cm_v2_p2.dep
+#include -c emp-fwk:boards/apollo/cm_v2/vu13p cm_v2_p1_2.dep device.dep packages_cm_v2_p1.dep
```

Comment:
@mcoshiro Based on the comment at the top of this file, this line wouldn't work due to unresolved dependencies. So should it be removed, or will it be relevant when we eventually move to TCDS2?